src/api/spec/features/webui/staging/projects_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'browser_helper'
```

Comment:
A feature spec to test behavior of a `before_action` is a bit overkill. Have a look at [`use_before_action`](http://matchers.shoulda.io/docs/v4.1.2/Shoulda/Matchers/ActionController.html#use_before_action-instance_method) like we do in the other controller tests.

Reply:
@hennevogel I removed the feature test, I know it was a bit overkill for this scenarion, was more the idea of getting the feature test's started for the staging. But I might tackle this in another PR, with things that a more worth to test.
This is kind of hard to test in the controller, ```use_before_action``` seems to check for the ```before_action``` globally in the controller which of course is there, but I can't test for the ```except``` of a certain method. Since we are not really testing for this in other controller test's, I would just leave it out.